contentcuration/contentcuration/frontend/channelEdit/components/sidePanels/SubmitToCommunityLibrarySidePanel/composables/useSpecialPermissions.js

```diff
@@ -23,7 +23,7 @@ const ITEMS_PER_PAGE = 3;
  *   Reactive state for the fetched, flattened permissions and pagination
  *   helpers used by `SpecialPermissionsList.vue`.
  */
-export function useSpecialPermissions(permissionIds) {
+export function useSpecialPermissions(channelVersionId, permissionIds) {
```

Comment:
Here we don't actually need the `permissionIds` anymore, since now the only identifier needed to fetch these special permissions will be the channelVersionId.